spec/views/post_index_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index page', type: :feature do
+  before(:each) do
+    @user = User.create(name: 'Abdul', bio: 'Aspiring FullStack Dev', photo: 'https://unsplash.com/photos/NDCy2-9JhUs',
+                        posts_counter: 2)
+    @user2 = User.create(name: 'Jonathan', bio: 'FullStack Dev', photo: 'https://unsplash.com/photos/hodKTZow_Kk',
+                         posts_counter: 2)
+    @post1 = Post.create(title: 'Test', text: 'First Post', comments_counter: 0, likes_counter: 0, author: @user)
+    @post2 = Post.create(title: 'Working?', text: 'Second Post', comments_counter: 0, likes_counter: 0, author: @user)
+    @comment1 = Comment.create(text: 'First Comment', post: @post1, author: @user2)
+    @comment2 = Comment.create(text: 'Second Comment', post: @post2, author: @user2)
+    @like1 = Like.create(post: @post1, author: @user2)
+  end
+
+  describe 'the post index page' do
+    it 'displays the users profile picture' do
+      visit user_posts_path(@user)
+      expect(page).to have_css("img[src*='https://unsplash.com/photos/NDCy2-9JhUs']")
+    end
+
+    it 'displays the users username' do
+      visit user_posts_path(@user)
+      expect(page).to have_content('Abdul')
+    end
+
+    it 'displays the number of posts the user has written' do
+      visit user_posts_path(@user)
+      expect(page).to have_content('2')
+    end
+
+    it 'displays the post title' do
+      visit user_posts_path(@user)
```

Comment:
- [ ] Your tests are failing please make sure to fix them so they are green in order to complete the requirements 
<img width="1083" alt="Screenshot 2023-02-01 at 16 14 09" src="https://user-images.githubusercontent.com/61790587/216082946-dd7c75f0-2982-4f78-8eb6-f334924e171e.png">

Reply:
Hello @youmari , 
Thanks for the review, however, the tests are running well on my end
As you have advised, I have attached a screenshot for the next reviewer

![image](https://user-images.githubusercontent.com/11190489/216088512-f2bd7001-4c7d-4997-bbd3-d446f9844955.png)